tests/DoctrineMongoODMModuleTest/Doctrine/CliTest.php

```diff
@@ -83,37 +86,17 @@ public function testValidHelpers()
 
     public function testValidCommands()
     {
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\GenerateDocumentsCommand',
-            $this->cli->get('odm:generate:documents')
-        );
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\GenerateHydratorsCommand',
-            $this->cli->get('odm:generate:hydrators')
-        );
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\GenerateProxiesCommand',
-            $this->cli->get('odm:generate:proxies')
-        );
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\GenerateRepositoriesCommand',
-            $this->cli->get('odm:generate:repositories')
-        );
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\QueryCommand',
-            $this->cli->get('odm:query')
-        );
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\Schema\CreateCommand',
-            $this->cli->get('odm:schema:create')
-        );
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\Schema\UpdateCommand',
-            $this->cli->get('odm:schema:update')
-        );
-        $this->assertInstanceOf(
-            'Doctrine\ODM\MongoDB\Tools\Console\Command\Schema\DropCommand',
-            $this->cli->get('odm:schema:drop')
+        self::assertInstanceOf(GenerateDocumentsCommand::class, $this->cli->get('odm:generate:documents'));
+        self::assertInstanceOf(GenerateHydratorsCommand::class, $this->cli->get('odm:generate:hydrators'));
+        self::assertInstanceOf(GenerateProxiesCommand::class, $this->cli->get('odm:generate:proxies'));
+        self::assertInstanceOf(GenerateRepositoriesCommand::class, $this->cli->get('odm:generate:repositories'));
+        self::assertInstanceOf(
+            GeneratePersistentCollectionsCommand::class,
+            $this->cli->get('odm:generate:persistent-collections')
         );
+        self::assertInstanceOf(QueryCommand::class, $this->cli->get('odm:query'));
+        self::assertInstanceOf(CreateCommand::class, $this->cli->get('odm:schema:create'));
+        self::assertInstanceOf(UpdateCommand::class, $this->cli->get('odm:schema:update'));
+        self::assertInstanceOf(DropCommand::class, $this->cli->get('odm:schema:drop'));
```

Comment:
Why `$this->` is changed to `self::`? I know it is also right, but I'd go with `$this->` because:
1. It is like that in the whole library - so to keep consistency
2. Even in PHPUnit documentation all examples are using `$this->`
3. PHPStorm bring back this functionality in latest release for PHPUnit usages